Player/Player.gd

```diff
@@ -2,26 +2,35 @@ extends CharacterBody2D
 
 signal interacted_with(interactable : Interactable)
 
-const SPEED = 50.0
+const ACCELERATION = 800
+const SPEED = 50
+const FRICTION = 300
```

Comment:
The `ACCELERATION` and `FRICTION` consts give our player a little ease-in and ease-out of their movement. It's slightly more natural but I tried not to make it too obvious. It shouldn't feel slippery.